modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -7,59 +7,29 @@
+
+When you create the service binding, the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
 
 .Prerequisites
 
 * You have created and deployed a Spring PetClinic sample application by using the *Developer* perspective.
 * You have installed {servicebinding-title} from the OperatorHub.
 * You have installed the *Crunchy Postgres for Kubernetes* Operator from the OperatorHub by using the `v5` *Update* channel.
-* You have created and deployed a Crunchy PostgreSQL database instance by using the *Developer* perspective. This instance has the following components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and `hippo-pgbouncer`.
+* You have created and deployed a Crunchy PostgreSQL database instance by creating  a *PostgresCluster* resource using the *Developer* perspective. This instance has the following components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and `hippo-pgbouncer`.
+
+.Binding connector
+image::odc-binding-connector.png[]
 
 .Procedure
 
 . Switch to the *Developer* perspective and ensure that you are in the appropriate project, for example, `my-petclinic`. 
 . In the *Topology* view, hover over the Spring PetClinic sample application to see a dangling arrow on the node.
 . Click and drag the arrow towards the *hippo* database in the Postgres Cluster to make a binding connection with the Spring PetClinic sample application. 
 
-.. Enter the name and click *Create*.
+.. In the *Create Service Binding* dialog, either keep the default name or add a different name for the service binding, and click *Create*.
 +
 .Service Binding dialog
 image::odc-sbc-modal.png[]
-
-
-Alternatively, in the *+Add* view, click the *YAML* option to see the *Import YAML* screen. Use the YAML editor and add the `ServiceBinding` resource:
-
-[source,YAML]
-----
-apiVersion: binding.operators.coreos.com/v1alpha1
-kind: ServiceBinding
-metadata:
-    name: spring-petclinic-pgcluster
-    namespace: my-petclinic
-spec:
-    services:
-    - group: postgres-operator.crunchydata.com
-      version: v1beta1
-      kind: PostgresCluster
-      name: hippo
-    application:
-      name: spring-petclinic
-      group: apps
-      version: v1
-      resource: deployments
-----
-
-A service binding request is created and the {servicebinding-title} controller projects the database service connection information into the application deployment as files by using a volume mount. After the request is successful, the application is redeployed and the connection is established.
```

Comment:
Is there any verification step?

Reply:
@max-cx Appreciate asking this question. Thanks!

@Dhruv-Soni11 See if you can use the content from step 3 of the last procedure from [here](https://docs.google.com/document/d/1mZ2FsOyc6orU7YdmuXm1jQYnNJtoIp3Xykl4xDqLyXY/edit#heading=h.yuxyyz8paahh). Please test it in the UI form your side. Hope it helps. Thanks!

Reply:
@Srivaralakshmi - I have never tested the UI before with respect to this procedure. Can you help me with the same? The reason is I am unsure if I have to just run the single procedure or run a number of procedures before that to get to the current procedure.

Reply:
@Dhruv-Soni11 sure, will do. We will meet tomorrow and sort this out. Thanks!

Reply:
Hi @Srivaralakshmi, I had a chat with @sanketpathak in this regard and In the UI you can check directly the status of an established service binding is connected if it is a positive connection.

Reply:
@max-cx - After a discussion with @Srivaralakshmi I had a few days ago, I have made some changes to the content as per her suggestions. Now the content looks more accurate from the completeness perspective and it also includes some other image updates suggested by her. The changes include some content addition at the end of the"**Creating a binding connection between components**" section to make sure it is now complete as well changes in the secondary section "**Adding a Service Binding connection in the YAML view**" as per your feedback.

I would request you to look at the content as I am positive that it would address your feedback.